src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockTSV.scala

```diff
@@ -107,8 +107,8 @@ object MockTSVStrings {
     List("sset_01", "sample_02").tabbed).newlineSeparated
 
   //entity TSVs
-  val entityUnknownFirstColumnHeader = List(
-    List("entity:sampel_id", "bar", "baz").tabbed,
+  val entityNonModelFirstColumnHeader = List(
+    List("entity:bigQueyr_id", "bar", "baz").tabbed,
```

Comment:
Typo here. Not sure what this means about the validity of the test.

Reply:
it's a typo on purpose. it fails with the firecloud model when calling import but will succeed with the flexible model because it doesn't care what the type is